### PR TITLE
Fix libvirt image variable

### PR DESCRIPTION
**Pull request by slagle**
- Drop container image ansible vars from samples
- Fix libvirt image variable

### CODE REVIEW

File: config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml [hunk 1]
```diff
@@ -125,15 +125,6 @@ spec:
...
          dns_search_domains: []
-         registry_url: quay.io/podified-antelope-centos9
-         image_tag: current-podified
-         image_prefix: openstack
-         edpm_ovn_controller_agent_image: "{{ registry_url }}/{{ image_prefix }}-ovn-controller:{{ image_tag }}"
-         edpm_iscsid_image: "{{ registry_url }}/{{ image_prefix }}-iscsid:{{ image_tag }}"
-         edpm_logrotate_crond_image: "{{ registry_url }}/{{ image_prefix }}-cron:{{ image_tag }}"
-         edpm_nova_compute_container_image: "{{ registry_url }}/{{ image_prefix }}-nova-compute:{{ image_tag }}"
-         edpm_nova_libvirt_container_image: "{{ registry_url }}/{{ image_prefix }}-nova-libvirt:{{ image_tag }}"
-         edpm_neutron_metadata_agent_image: "{{ registry_url }}/{{ image_prefix }}-neutron-metadata-agent-ovn:{{ image_tag }}"
```
Comment (bshephar) [inline config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml:136]: Yeah, I was wondering if all of these had a practical purpose. Good to drop them.

I thought maybe the ci-framework was using kustomize to replace the URL's for internal, downstream testing.

**Review by bshephar — COMMENTED** [review]
Kuttl test failure seems unrelated

**Review by gibizer — APPROVED** [review]

**Comment by gibizer** [comment]
waiting for kuttl to be grene

**Comment by abays** [comment]
> @slagle: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> ci/prow/dataplane-operator-build-deploy-kuttl 	[b999680](https://github.com/openstack-k8s-operators/dataplane-operator/commit/b99968024e19bed30becc14fbb1c4e650df98d3d) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_dataplane-operator/512/pull-ci-openstack-k8s-operators-dataplane-operator-main-dataplane-operator-build-deploy-kuttl/1724017353861107712) 	true 	`/test dataplane-operator-build-deploy-kuttl`
> 
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openstack-k8s-operators&repo=dataplane-operator&pr=512). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:slagle).

This keeps failing with:

```
{  failed to wait for the created cluster claim to become ready: timed out waiting for the condition}
```

...which is not caused by these changes.

/test dataplane-operator-build-deploy-kuttl

**Comment by slagle** [comment]
/test dataplane-operator-build-deploy-kuttl

The kuttl tests pass:

`
INFO[2023-11-13T12:43:20Z] Running step dataplane-operator-build-deploy-kuttl-openstack-k8s-operators-kuttl. 
INFO[2023-11-13T12:48:40Z] Step dataplane-operator-build-deploy-kuttl-openstack-k8s-operators-kuttl succeeded after 5m19s. 
`

It's the deploy after that which fails:

`
 Failed to discover available identity versions when contacting https://keystone-public-openstack.apps.oko-01.oooci.ccitredhat.com. Attempting to parse version from URL.
Could not find versioned identity endpoints when attempting to authenticate. Please check that your auth_url is correct. Unable to establish connection to https://keystone-public-openstack.apps.oko-01.oooci.ccitredhat.com: HTTPSConnectionPool(host='keystone-public-openstack.apps.oko-01.oooci.ccitredhat.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError(": Failed to resolve 'keystone-public-openstack.apps.oko-01.oooci.ccitredhat.com' ([Errno -2] Name or service not known)"))
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"k8s.io/test-infra/prow/entrypoint/run.go:84","func":"k8s.io/test-infra/prow/entrypoint.Options.internalRun","level":"error","msg":"Error executing test process","severity":"error","time":"2023-11-13T13:00:11Z"}
error: failed to execute wrapped command: exit status 1 
INFO[2023-11-13T13:00:11Z] Step dataplane-operator-build-deploy-kuttl-openstack-k8s-operators-deploy failed after 11m31s. 
`

File: config/default/manager_default_images.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ spec:
           value: quay.io/podified-antelope-centos9/openstack-cron:current-podified
         - name: RELATED_IMAGE_OPENSTACK_EDPM_NOVA_COMPUTE_DEFAULT_IMG
           value: quay.io/podified-antelope-centos9/openstack-nova-compute:current-podified
-        - name: RELATED_IMAGE_OPENSTACK_EDPM_NOVA_LIBVIRT_DEFAULT_IMG
+        - name: RELATED_IMAGE_OPENSTACK_EDPM_LIBVIRT_DEFAULT_IMG
```
Comment (dprince) [inline config/default/manager_default_images.yaml:22]: If you change this ENV variable you will also need to adjust the name in SetupAnsibleImageDefaults() function within controllers/openstackdataplanenodeset_controller.go

**Review by dprince — CHANGES_REQUESTED** [review]

File: config/default/manager_default_images.yaml (hunk 2 above)
Comment (slagle) [reply config/default/manager_default_images.yaml:22]: thanks. I missed that initially.

**Comment by slagle** [comment]
recheck

```
2023-11-14 10:25:46.307924 | controller | TASK [edpm_deploy_baremetal : Wait for baremetal nodes to reach 'provisioned' state _raw_params=oc wait bmh --all -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }} --for=jsonpath='{.status.provisioning.state}'=provisioned --timeout={{ cifmw_edpm_deploy_baremetal_wait_bmh_timeout_mins }}m] ***
2023-11-14 10:25:46.307940 | controller | Tuesday 14 November 2023  10:25:46 -0500 (0:00:57.638)       0:22:50.968 ******
2023-11-14 10:45:46.667253 | controller | fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["oc", "wait", "bmh", "--all", "-n", "openstack", "--for=jsonpath={.status.provisioning.state}=provisioned", "--timeout=20m"], "delta": "0:20:00.128865", "end": "2023-11-14 10:45:46.624993", "msg": "non-zero return code", "rc": 1, "start": "2023-11-14 10:25:46.496128", "stderr": "error: timed out waiting for the condition on baremetalhosts/edpm-compute-0", "stderr_lines": ["error: timed out waiting for the condition on baremetalhosts/edpm-compute-0"], "stdout": "", "stdout_lines": []}
```

**Review by dprince — APPROVED** [review]

**Comment by slagle** [comment]
recheck

**Comment by slagle** [comment]
recheck

**Comment by abays** [comment]
> Build failed (check pipeline). Post `recheck` (without leading slash) to rerun all jobs. Make sure the failure cause has been resolved before you rerun jobs.
> 
> https://review.rdoproject.org/zuul/buildset/89484821e8f544b39419b09f342eb41e
> 
> ✔️ [openstack-k8s-operators-content-provider ](https://review.rdoproject.org/zuul/build/5dda7621896f4ba28f4a94604225bbf7) SUCCESS in 1h 23m 30s ✔️ [podified-multinode-edpm-deployment-crc ](https://review.rdoproject.org/zuul/build/c5fe7503410945999ce5f049f2c0b6ba) SUCCESS in 1h 02m 29s ❌ [cifmw-crc-podified-edpm-baremetal ](https://review.rdoproject.org/zuul/build/d4491d9ce1c44e57875d92e0dc7182a1) **FAILURE** in 1h 05m 54s ✔️ [dataplane-operator-docs-preview ](https://review.rdoproject.org/zuul/build/0f7db05e69dc42658961ca3e52a05e24) SUCCESS in 1m 52s

The baremetal Zuul job seems to fail a lot across multiple runs for multiple PRs in various repos before finally passing on the Nth try:

```
TASK [edpm_deploy_baremetal : Wait for baremetal nodes to reach 'provisioned' state _raw_params=oc wait bmh --all -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }} --for=jsonpath='{.status.provisioning.state}'=provisioned --timeout={{ cifmw_edpm_deploy_baremetal_wait_bmh_timeout_mins }}m] ***
```

It seems like it might be worth investigating further.

**Comment by rabi** [comment]
> It seems like it might be worth investigating further.

I had looked at it and could possibly be fixed with https://github.com/openstack-k8s-operators/install_yamls/pull/627

**Pull request merged**